tools/pagerduty-migrator/migrator/resources/integrations.py

```diff
@@ -43,9 +44,14 @@ def migrate_integration(integration: dict, escalation_policies: list[dict]) -> N
 
     oncall_name = "{} - {}".format(integration["service"]["name"], integration["name"])
 
+    if "oncall_type" in integration.keys():
+        oncall_type = integration["oncall_type"]
+    else:
+        oncall_type = "webhook"
```

Comment:
Not really sure about falling back to the "webhook" integration type, probably we could make this as a feature flag in an env variable? Something like `FALLBACK_TO_WEBHOOK_INTEGRATION_TYPE` and make this as a separate PR? The idea behind the integration type matching inititally was to match them 1 to 1 without any guessing or fallbacks.